WeTransferPRLinter/Sources/WeTransferPRLinter/XCResultReporting/XCResultCoverageFactory.swift

```diff
@@ -24,7 +24,9 @@ struct XCResultCoverageReporter {
             }
 
             let testSummaries = invocationRecord.actions.map { $0.testPlanRunSummaries(resultFile: resultFile) }
-            let testTargetNames = testSummaries.compactMap { $0?.summaries.flatMap { $0.testableSummaries.compactMap(\.targetName) }}.flatMap { $0 }
+            let testTargetNames = testSummaries.compactMap {
+                $0?.summaries.flatMap { $0.testableSummaries.compactMap(\.targetName) }
+            }.flatMap { $0 }
```

Comment:
If it fails, I wonder if this is the line that is causing an issue.

```suggestion
            }
            .flatMap { $0 }
```

And if it is, how we could improve it.

Reply:
I left this out of https://github.com/WeTransfer/WeTransfer-iOS-CI/pull/157/commits/1a9effa637ae855a4f90eee879a3c0e65f0c0e6c in https://github.com/WeTransfer/WeTransfer-iOS-CI/pull/157 to check it out.